sql/README.md

```diff
@@ -54,58 +58,53 @@ Commas are placed at the end of the column name.
 
 ```
 -- =============================================
--- Author:      [Name]
+-- Author:		[Name]
 -- Create date: [mm/dd/yyyy]
 -- Description:	[add description]
 -- Update Log:
--- VERSION  DATE        AUTHOR     DESCRIPTION
--- V#       yyyy-mm-dd	<name>     <What changed?>
+--	VERSION		DATE			AUTHOR		DESCRIPTION
+--	 V#			 yyyy-mm-dd		 <name>		 <What changed?>
 -- =============================================
 CREATE PROCEDURE [dbo].[usp_fuel_products_get_by_id] 
-  @id INT
+	@id INT
 AS
 BEGIN
-  SET NOCOUNT ON;
-
-  SELECT id,
-    display_name,
-    fuel_group_id,
-    octane_id,
-    ethanol_id,
-    category_id
-  FROM
-    [dbo].[fuel_products] WITH(NOLOCK)
-  WHERE
-    id = @id;
+	SET NOCOUNT ON;
+
+	SELECT		id,
+				display_name,
+				fuel_group_id,
+				octane_id,
+				ethanol_id,
+				category_id
+	FROM 		[dbo].[fuel_products] WITH(NOLOCK)
+	WHERE 		id = @id;
+
 END
```

Comment:
I hadn't noticed the changes in the select format, I will leave the merge to @dubgb and @normrider2001